features/netsocket/emac-drivers/TARGET_STM_EMAC/stm32xx_emac.cpp

```diff
@@ -475,6 +817,8 @@ void mbed_default_mac_address(char *mac)
     uint32_t word0 = *(uint32_t *)0x1FFF7A10;
 #elif defined (TARGET_STM32F7)
     uint32_t word0 = *(uint32_t *)0x1FF0F420;
+#elif defined (TARGET_STM32H7)
+    uint32_t word0 = *(uint32_t *)0x1FF0F420;
```

Comment:
Value to check

Reply:
Replaced with correct value (0x1FF1E800)